### DISCUSSION TITLE
Followed artists monthly digest email?

### DISCUSSION THREAD

**Discussion by emptyvesselnz** [funmusicplace/mirlo#239] [Ideas]
One thing that occurred to me a while ago: I never ever keep up with checking my BC emails because there are way too many. 
A "digest" email once a month would be so great even in quite a basic form: "Here's the roundup of releases this month for artists you follow"
maybe a dumb idea but I'd like it and maybe others would. So I could opt out of getting the flood of emails the labels and artists wants to send me twice a week which I don't read but I could still keep up with what's going on and if there's anything I'd like to buy.

**Comment by lukaprincic** [comment]
this is a great idea.
